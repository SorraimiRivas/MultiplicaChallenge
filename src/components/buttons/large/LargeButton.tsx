import {View, Text, TouchableOpacity} from 'react-native';
import React from 'react';
import styles from './styles';

const LargeButton = ({title}) => {
  return (
    <TouchableOpacity style={styles.container}>
      <Text style={styles.text}>{title}</Text>
    </TouchableOpacity>
  );
};

export default LargeButton;
