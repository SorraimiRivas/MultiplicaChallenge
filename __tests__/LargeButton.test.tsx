import React from 'react';
import renderer from 'react-test-renderer';
import LargeButton from '../src/components/buttons/large/LargeButton';

test('renders correctly', () => {
  const tree = renderer
    .create(<LargeButton title="Test" onPress={() => null} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
