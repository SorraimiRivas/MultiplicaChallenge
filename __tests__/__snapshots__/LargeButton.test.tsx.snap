// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#334FFA",
      "borderRadius": 10,
      "height": 50,
      "marginTop": 40,
      "opacity": 1,
      "padding": 14,
      "width": 353,
    }
  }
>
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontFamily": "Avenir",
        "fontSize": 16,
        "fontWeight": "800",
      }
    }
  >
    Test
  </Text>
</View>
`;
